.evergreen/run-mongodb-oidc-test.sh

```diff
@@ -13,20 +13,21 @@ fi
 
 # Get the drivers secrets.  Use an existing secrets file first.
 if [ ! -f "./secrets-export.sh" ]; then
-    bash .evergreen/tox.sh -m aws-secrets -- drivers/oidc
```

Comment:
What's the rationale behind directly calling `setup_secrets.sh` instead of using tox?

Reply:
Our tox env replicates the env requirements in DRIVERS_TOOLS, I think we should migrate away from it.

Reply:
If we migrate away from the tox env, what process do we envision for running these tests locally?

Reply:
Good point! Can we add that step to the local testing documentation?

Reply:
I opened https://jira.mongodb.org/browse/PYTHON-3986